Jenkinsfile

```diff
@@ -93,8 +93,8 @@ pipeline {
     stage('L0: Unit Tests CPU') {
       when {
         anyOf {
-          branch 'main'
-          changeRequest target: 'main'
+          branch'main'
+          changeRequest target:'main'
```

Comment:
please revert if you did not make any necessary changes for Jenkins? It would block all CIs. There are many similar changes to be reverted in this file.